spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
+    before :each do
+      Division.delete_all
 
+      @division = Division.create(id: 5, name: "hey jude", date: Date.new(2016,12,25),
+      number: 1, house: "representatives", source_url: "", debate_url: "", motion: "",
+      source_gid: "", debate_gid: "")
+
+      @other_division = Division.create(id: 6, name: "beautiful boy", date: Date.new(2016,06,01),
+      number: 1, house: "representatives", source_url: "", debate_url: "", motion: "",
+      source_gid: "", debate_gid: "")
+    end
+
+    context "when request has no parameters" do
+      it "should return index page with all divisions" do
```

Comment:
Actually, this is not accurate. The code for that is this:

```ruby
@years = (Division.order(:date).first.date.year..Division.order(:date).last.date.year).to_a
# lots 
# of 
# code 
# here...

# Set the year to the lastest we have data for if it's not set
@year = @years.last if @rdisplay.nil? && @date.nil? && @month.nil? && @year.nil?
```

Which in production, it will happens as you say as long as we keep adding divisions. But the difference is important to do my testing, because that way I don't need to mock the System date :)